execution.bs

```diff
@@ -6126,48 +6128,55 @@ template&lt;class Domain, class Tag, sender Sndr, class... Args>
               [&]&lt;class Sch, class Child>(auto, Sch sch, Child&& child) {
                 using <i>variant-type</i> = <i>see below</i>;
                 using <i>receiver-type</i> = <i>see below</i>;
+                using <i>operation-type</i> = connect_result_t&lt;schedule_result_t&lt;Sch>, <i>receiver-type</i>>;
 
                 struct <i>state-type</i> {
-                  Rcvr& rcvr;
-                  <i>variant-type</i> async_result;
-                  connect_result_t&lt;schedule_result_t&lt;Sch>, <i>receiver-type</i>> op_state2;
+                  Rcvr& <i>rcvr</i>;
+                  <i>variant-type</i> <i>async-result</i>;
+                  <i>operation-type</i> <i>op-state</i>;
 
                   explicit <i>state-type</i>(Sch sch, Rcvr& rcvr)
-                    : rcvr(rcvr), op_state2(connect(schedule(sch), <i>receiver-type</i>{<i></i>{}, this})) {}
+                    : <i>rcvr</i>(rcvr), <i>op-state</i>(connect(schedule(sch), <i>receiver-type</i>{<i></i>{}, this})) {}
                 };
+
                 return <i>state-type</i>{sch, rcvr};
               },
               std::forward&lt;Sndr>(sndr));
           }
           </pre>
 
-          1. Let `Sigs` be a pack of the arguments to the
+          1. The local class <i>`state-type`</i> is a structural type suitable
+              for use with structured bindings. Its data members are
+              exposition-only.
```

Comment:
```suggestion
          1. The local class <i>`state-type`</i> is a structural type.
```

I think this is sufficient, and avoids the risk of hostile readings of "its data members are exposition-only" (what is exposition only about them? Just their names? Their order? Their types?). If we just say it's a structural type then that means you can rely on members of those type in that order, and no other members.